docs/code-reviews/PR1/27b99435126e3d7a58706a4f6e0d20a5c02b1608.md

```
@@ -0,0 +1,287 @@
+<!-- internal state end -->
+```
+
+_Meta_: https://github.com/flyingrobots/draft-punks/pull/1#issuecomment-3344395219
+
+> [!note]- **Accepted**
+> | Confidence | Remarks |
+> |------------|---------|
+> | 0 | This seems like a bug... |
+>
+> ## Lesson Learned
+> 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)